examples/quickstart.yaml

```diff
@@ -48,11 +48,23 @@ data:
     apiHost: https://api.honeycomb.io/
     watchers:
       - dataset: kubernetes-logs
-        labelSelector: component=kube-controller-manager,tier=control-plane
+        labelSelector: "component=kube-apiserver,tier=control-plane"
         namespace: kube-system
         parser: glog
       - dataset: kubernetes-logs
-        labelSelector: component=kube-scheduler,tier=control-plane
+        labelSelector: "component=kube-scheduler,tier=control-plane"
+        namespace: kube-system
+        parser: glog
+      - dataset: kubernetes-logs
+        labelSelector: "component=kube-controller-manager,tier=control-plane"
+        namespace: kube-system
+        parser: glog
+      - dataset: kubernetes-logs
+        labelSelector: "k8s-app=kube-proxy"
```

Comment:
is it correct that this still says k8s-app even though the rest have been changed to app= or component=?

Reply:
yes, core components from K8s have a `k8s-app` label.